test/scenarios.py

```diff
@@ -6766,6 +6766,132 @@
             ),
         },
     ),
+    Scenario(
+        "scenario31-3",
+        "Remove file on sending side without rejecting it first. Expect error and file present in JSON output",
```

Comment:
Can you clarify the description here that it's about removing the file from the DB, not physically from the disk?

Reply:
Sure! I've clarified that it's about removal from the transfer itself